--- a/test/testcases/edit.exp	2016-12-09 13:28:17.941377888 +0000
+++ b/test/testcases/edit.exp	2016-12-09 13:28:22.777899242 +0000
@@ -83,3 +83,6 @@
 ERROR: 46: g1 refers to missing object nosuch
-.INP: filter "sed 's/^/# this is a comment\n/'" loc-d1
+.INP: filter "sed 's/^/# this is a comment
+ERROR: 47: configure.modgroup: No closing quotation
+.INP: /'" loc-d1
+ERROR: 48: configure.modgroup: No closing quotation
 .INP: rsc_defaults $id="rsc_options" failure-timeout=10m
