--- a/test/testcases/edit.exp	2016-10-20 22:50:33.744542999 +0000
+++ b/test/testcases/edit.exp	2016-10-20 22:50:39.591102718 +0000
@@ -83,3 +83,6 @@
 ERROR: 46: g1 refers to missing object nosuch
-.INP: filter "sed 's/^/# this is a comment\n/'" loc-d1
+.INP: filter "sed 's/^/# this is a comment
+ERROR: 47: configure.modgroup: No closing quotation
+.INP: /'" loc-d1
+ERROR: 48: configure.modgroup: No closing quotation
 .INP: rsc_defaults $id="rsc_options" failure-timeout=10m
--- a/test/testcases/resource.exp	2016-10-20 22:50:33.768542999 +0000
+++ b/test/testcases/resource.exp	2016-10-20 22:51:02.357001702 +0000
@@ -136,4 +136,2 @@
 .EXT crm_resource -r 'p0' -p 'a0' -v '1 2 3'
-
-Set 'p0' option: id=p0-instance_attributes-a0 set=p0-instance_attributes name=a0=1 2 3
 .INP: configure
@@ -181,3 +179,3 @@
 .EXT crm_resource -r 'p0' -d 'a0'
-Deleted 'p0' option: id=p0-instance_attributes-a0 name=a0
+Deleted p0 option: id=p0-instance_attributes-a0 name=a0
 .INP: configure
@@ -201,4 +199,2 @@
 .EXT crm_resource --meta -r 'p0' -p 'm0' -v '123'
-
-Set 'p0' option: id=p0-meta_attributes-m0 set=p0-meta_attributes name=m0=123
 .INP: configure
@@ -248,3 +244,3 @@
 .EXT crm_resource --meta -r 'p0' -d 'm0'
-Deleted 'p0' option: id=p0-meta_attributes-m0 name=m0
+Deleted p0 option: id=p0-meta_attributes-m0 name=m0
 .INP: configure
@@ -922,3 +918,2 @@
 .EXT crm_simulate -sUL
-3 of 6 resources DISABLED and 0 BLOCKED from being started due to failures
 
@@ -932,3 +927,3 @@
      Stopped: [ node1 ]
- p3	(ocf::heartbeat:Dummy):	Stopped ( disabled ) 
+ p3	(ocf::heartbeat:Dummy):	Stopped 
  Master/Slave Set: msg [g]
--- a/test/testcases/ra.exp	2016-10-20 22:50:33.764542999 +0000
+++ b/test/testcases/ra.exp	2016-10-20 22:51:07.084514246 +0000
@@ -24,8 +24,5 @@
 
-state (string, [/var/run/Dummy-Dummy.state]): State file
+state (string, [/var/run//Dummy-{OCF_RESOURCE_INSTANCE}.state]): State file
     Location to store the resource state in.
 
-passwd (string): Password
-    Fake password field
-
 fake (string, [dummy]): 
@@ -37,10 +34,2 @@
 
-fail_start_on (string): Report bogus start failure on specified host
-    Start actions will return failure if running on the host specified here, but
-    the resource will start successfully anyway (future monitor calls will find it
-    running). This can be used to test on-fail=ignore.
-
-envfile (string): Environment dump file
-    If this is set, the environment will be dumped to this file for every call.
-
 Operations' defaults (advisory minimum):
@@ -74,2 +63,5 @@
 
+stonith-timeout (time, [60s]): How long to wait for the STONITH action to complete per a stonith device.
+    Overrides the stonith-timeout cluster property
+
 priority (integer, [0]): The priority of the stonith resource. Devices are tried in order of highest priority to lowest.
@@ -87,10 +79,2 @@
 
-pcmk_delay_max (time, [0s]): Enable random delay for stonith actions and specify the maximum of random delay
-    This prevents double fencing when using slow devices such as sbd.
-    Use this to enable random delay for stonith actions and specify the maximum of random delay.
-
-pcmk_action_limit (integer, [1]): The maximum number of actions can be performed in parallel on this device
-    Pengine property concurrent-fencing=true needs to be configured first.
-    Then use this to specify the maximum number of actions can be performed in parallel on this device. -1 is unlimited.
-
 pcmk_reboot_action (string, [reboot]): Advanced use only: An alternate command to run instead of 'reboot'
--- a/test/testcases/newfeatures.exp	2016-10-20 22:50:33.756542999 +0000
+++ b/test/testcases/newfeatures.exp	2016-10-20 22:51:11.803701832 +0000
@@ -18,2 +18,3 @@
 .INP: alert notify_9 /usr/share/pacemaker/alerts/alert_snmp.sh         attributes         trap_add_hires_timestamp_oid="false"         trap_node_states="non-trap"         trap_resource_tasks="start,stop,monitor,promote,demote"         to "192.168.40.9"
+ERROR: 17: alert not supported by the RNG schema
 .INP: show tag:ones and type:location
@@ -44,5 +45,2 @@
 	stonith-enabled=no
-alert notify_9 "/usr/share/pacemaker/alerts/alert_snmp.sh" \
-	attributes trap_add_hires_timestamp_oid=false trap_node_states=non-trap trap_resource_tasks="start,stop,monitor,promote,demote" \
-	to 192.168.40.9
 .INP: _test
